qiskit/transpiler/_transpiler.py

```diff
@@ -21,35 +19,28 @@
 from qiskit.mapper import (Coupling, optimize_1q_gates, coupling_list2dict, swap_mapper,
                            cx_cancellation, direction_mapper,
                            remove_last_measurements, return_last_measurements)
-from qiskit.qobj import Qobj, QobjConfig, QobjExperiment, QobjItem, QobjHeader
 from ._parallel import parallel_map
 
 logger = logging.getLogger(__name__)
 
 
 # pylint: disable=redefined-builtin
-def compile(circuits, backend,
-            config=None, basis_gates=None, coupling_map=None, initial_layout=None,
-            shots=1024, max_credits=10, seed=None, qobj_id=None, hpc=None,
-            pass_manager=None):
+def compile(circuits, backend, basis_gates=None, coupling_map=None, initial_layout=None,
+            seed_mapper=None, hpc=None, pass_manager=None):
     """Compile a list of circuits into a qobj.
 
     Args:
         circuits (QuantumCircuit or list[QuantumCircuit]): circuits to compile
         backend (BaseBackend): a backend to compile for
-        config (dict): dictionary of parameters (e.g. noise) used by runner
         basis_gates (str): comma-separated basis gate set to compile to
         coupling_map (list): coupling map (perhaps custom) to target in mapping
         initial_layout (list): initial layout of qubits in mapping
-        shots (int): number of repetitions of each circuit, for sampling
-        max_credits (int): maximum credits to use
-        seed (int): random seed for simulators
-        qobj_id (int): identifier for the generated qobj
+        seed_mapper (int): random seed for the swap_mapper
         hpc (dict): HPC simulator parameters
         pass_manager (PassManager): a pass_manager for the transpiler stage
 
     Returns:
-        QobjExperiment: Experiment to be wrapped in a Qobj.
+        dags: a list of dags.
```

Comment:
Shouldn't `compile` continue returning QobjExperiments?

Reply:
this is a mistake it should be dag and circuits in the future